data/json/mutations.json

```diff
@@ -3925,7 +3925,11 @@
         "description" : "Your body's ability to digest fruits, vegetables and grains is severely hampered.  You cannot eat anything besides meat.",
         "cancels" : ["VEGETARIAN", "HERBIVORE", "RUMINANT",  "GRAZER"],
         "leads_to" : ["SAPROVORE"],
-        "category" : ["MUTCAT_LIZARD", "MUTCAT_BEAST", "MUTCAT_SPIDER", "MUTCAT_CHIMERA", "MUTCAT_RAPTOR", "MUTCAT_FELINE"]
+        "category" : ["MUTCAT_LIZARD", "MUTCAT_BEAST", "MUTCAT_SPIDER", "MUTCAT_CHIMERA", "MUTCAT_RAPTOR", "MUTCAT_FELINE"],
+        "vitamin_rates": [
+            [ "VitA", -999 ],
+            [ "VitC", -999 ]
```

Comment:
> Doesn't seem to work, I set vitamin C to -1000, debugged in carnivore, then it dropped to -200 and stayed there.

-1000 is outside of the supported range for that vitamin?

> Also, consider hypervitaminosis and negative rate, specifically the case where a survivor with hypervitaminosis gains a mutation that generates said vitamins.

Mutation traits and bionics needs addressing as a whole in a further PR